### PR TITLE
Vote card minor fixes

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
Ok for me :+1:

File: src/Components/VoteCard/index.jsx [hunk 1]
```diff
@@ -45,8 +45,10 @@ const VoteCard = ({ ...car }) => {
         <div className="imageContainer">
           <img src={car.image} alt="VW Golf 1 GTI" />
         </div>
-        <h1 className="titleArea">{car.model}</h1>
-        <p className="priceArea">A partir de {car.price}€</p>
+        <h1 className="titleArea">
+          {car.brand} {car.model}
+        </h1>
+        <p className="PriceArea">A partir de {car.price}€</p>
```
Comment (Hacene010) [inline src/Components/VoteCard/index.jsx]: className in camelCase plz

File: src/Pages/Home/index.jsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ export default function Home() {
 
   useEffect(() => {
     axios
-      .get(`/home`)
+      .get(`http://localhost:3031/news`)
```
Comment (Hacene010) [inline src/Pages/Home/index.jsx:55]: Don't forget to hide your localhost into a .env file

**Review by Hacene010 — CHANGES_REQUESTED** [review]

File: src/Pages/Home/index.jsx (hunk 2 above)
Comment (NacymeG) [reply src/Pages/Home/index.jsx:55]: Next PR

**Review by Caroline-CD — APPROVED** [review]
Ok

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**